src/pages/MemberSettingsPage/MembershipTierPage/MembershipTier.jsx

```diff
@@ -0,0 +1,134 @@
+import * as React from "react";
+import { Box, Typography, Button, Stack } from "@mui/material";
+import CheckIcon from "@mui/icons-material/Check";
+
+const MemberShipTier = () => {
+  return (
+    <Box m="20px">
+      <Typography sx={{ fontWeight: "bold" }} variant="h2" marginBottom="20px">
+        Choose Your Plan
+      </Typography>
+
+      <Box
+        display="grid"
+        gridTemplateColumns="repeat(3, 1fr)"
+        gap="20px"
+        justifyContent="center"
+      >
+        <Box
+          p="20px"
+          borderRadius="10px"
+          fontWeight="bold"
+          boxShadow="0 4px 10px rgba(0,0,0 0.1)"
+          bgcolor="#555555"
+        >
+          <Typography sx={{ fontWeight: "bold" }} variant="h3" mb="20px">
+            Basic Plan
+          </Typography>
+          <Typography
+            sx={{ fontWeight: "bold" }}
+            variant="h4"
+            color="#FFFFFF"
+            mb="20px"
+          >
+            $9.99/month
+          </Typography>
+          <Box
+            sx={{ display: "flex", alignItems: "center", marginBottom: "10px" }}
+          >
+            <CheckIcon style={{ fontSize: "16px", marginRight: "8px" }} />
+            <Typography mb="2px">Add Up To 5 Services</Typography>
+          </Box>
+          <Box
+            sx={{ display: "flex", alignItems: "center", marginBottom: "10px" }}
+          >
+            <CheckIcon style={{ fontSize: "16px", marginRight: "8px" }} />
+            <Typography mb="2px">Client User Chat Feature</Typography>
+          </Box>{" "}
+          <Box
+            sx={{ display: "flex", alignItems: "center", marginBottom: "10px" }}
+          >
+            <CheckIcon style={{ fontSize: "16px", marginRight: "8px" }} />
+            <Typography mb="2px">Invoice Tracking</Typography>
+          </Box>
+          <Button variant="contained" color="primary">
+            Choose Plan
+          </Button>
+        </Box>
```

Comment:
Since the general structure is reusable code lets turn this into one component and have the props be the pricing details. Makes the code a lot cleaner and easier to maintain.